.travis.yml

```diff
@@ -49,5 +49,5 @@ jobs:
           tags: true
 env:
   global:
-  # GITHUB_RELEASE_TOKEN (2374......46ed)
-  - secure: Q0Nl6PYZhAvS2rUprp7yxAnFmaaWlRJ7Vek1y8uQ0h2Rm4hraRDTwmcFSKOdOgHnI75Wzn5qCcTDHnQMVu1tHUf7/Z1Fnk/2cG5Y5G/R877jihoaaSNdhlbBPV92EZ5+tSc51H3hmChoK4FePKv45KQLsz59txOYmsVzKd8ryKTXzrDSH5IA3ssyBLfRTT6RbRDdTRBOpoTR4zPBkWCj9vB1Pd4g01jZPk2oP1bd8sIfJp1vrEzRCOrR7FiuDIBNArKTqOoLt3rlNry4uZpwrZmmA5b0L7xNM1C1PUxMjCKZV7mT5MthVx9R2NVOrddxAE0P7z/6PK1V86oJYwoSySOedLM4AYCwddpZNzqIR5aEAM33rvbRYuyCURt/1w59SXb5zftigdjbVg7wbvxb7sxWOEbhg+LHlno0+8U1r2WU1OY3BZBvsEKCV4jGGJpSyhvQpJ0gyWfsNk3Kt37OFgG0ZBNdXgIfQ+jSzan8puoR8wUbz4DyuYOV/z5Y4LdswFR3q4/eo1FjjzBBz+fXDodXxUm2XT/lxz5uL23y5EUqfT6vIBXd2XhlQ/izS0PCckzPILSz+nOL/I4Hq/q2Zfzu+9kD/sL6G3n1c28LbQnNOsuXyfoBozVZvwPc8wXWf4jIJlWGUXCoyG2zRJkN5D64GnSDvzu6+TThOYU8ZNQ=
\ No newline at end of file
+  # GITHUB_RELEASE_TOKEN
```

Comment:
nit: Add token snippet here so we know which one it is (unless there is a reason you explicitly removed it)

Reply:
I removed it on purpose as GitHub doesn't expose the snippet -- they're named instead and this one is named `brightspace-ui-core` :)